### PR TITLE
Typo fixes

### PR DESCRIPTION
**Pull request by tipakA**

### CODE REVIEW

File: guide/preparations/README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ To install and use discord.js, you'll need to install it via npm \(Node's packag
 
 Like any other project, you should have a dedicated folder for this, in order to keep it organized and manageable.
 
-Navigate to a place on your machine where it's be easy for find and open in the future, for convenience purposes. Create a new folder like you normally would (for Linux, you can use `mkdir project-name` inside your terminal). If you already have a name you want to use for your bot, you can use that as the folder name. Otherwise, you may name it something like `discord-bot` for the time being \(or anything else you have in mind\).
+Navigate to a place on your machine where it's easy to find and open it in the future, for convenience purposes. Create a new folder like you normally would (for Linux, you can use `mkdir project-name` inside your terminal). If you already have a name you want to use for your bot, you can use that as the folder name. Otherwise, you may name it something like `discord-bot` for the time being \(or anything else you have in mind\).
```
Comment (almostSouji) [inline guide/preparations/README.md]: Imo that sounds as strange as the original
How about "Navigate to a place on your machine that's easy to find and reopen in the future"?

Comment (tipakA) [reply guide/preparations/README.md]: Yeah, that sounds a bit better

**Comment by Anish-Shobith** [comment]
Hmm, i was late but  in the same file in ur pr mostly a `,` should have been used instead of `;` https://hasteb.in/uhokazen.md after discord.js in this link i have posted

**Comment by tipakA** [comment]
According to [this site](https://writing.wisc.edu/handbook/grammarpunct/semicolons/), that is properly used semicolon, as both clauses are independent.

**Comment by Anish-Shobith** [comment]
Ahh, Thanks for bringing it into my notice i am gonna keep a note of it :+1:

**Pull request merged**